src/actors/MeleeMob.gd

```diff
@@ -54,6 +56,9 @@ func set_sigils(t, default) -> void:
 
 func trigger_death(drop_loot = true) -> void:
 	_drop_loot = drop_loot
+	Game.score += EXPERIENCE
+	print(get_parent().get_parent())
+	get_parent().get_parent().get_node("CanvasLayer/hud/HBoxContainer2/Experience").text = "Experience: " + str(Game.score)
```

Comment:
This can be updated via a signal or on the UI update tick in the game file.

Reply:
Definitely agree with the signal. I don't like this text updating on a tick because I think the delay for experience gain takes away from the immediate reward feeling a player gets after killing an enemy